src/test/java/org/ibp/api/java/impl/middleware/inventory/LocationValidatorTest.java

```diff
@@ -210,11 +237,236 @@ public void testvalidateInvalidLocation() {
 		this.errors = new MapBindingResult(new HashMap<String, String>(), LotUpdateRequestDto.class.getName());
 
 		try {
-			this.locationValidator.validateLocation(errors, LOCATION_ID);
+			this.locationValidator.validateLocation(errors, LocationValidatorTest.LOCATION_ID);
```

Comment:
Please take the chance to fix the name of the functions: 
testvalidateRequiredLocation
testvalidateInvalidLocation